src/test/java/lotto/domain/LottoNumberTest.java

```diff
@@ -38,7 +40,7 @@ public void findAllLottoNumbersMapToList() {
 
         //Then
         assertThat(result).isNotNull();
-        assertThat(result).hasSize(45);
+        assertThat(result).hasSize(MAX_LOTTO_NUMBER);
```

Comment:
만약 도메인 로직에서 로또 번호에 대한 요구사항이 바뀌게 되면 해당 테스트도 영향을 받게 되겠네요! 🤔 
```suggestion
        assertThat(result).hasSize(LottoNumber.MAX);
```